vic/drivers/cesm/bld/vic.buildexe.csh

```diff
@@ -0,0 +1,23 @@
+#! /bin/csh -f
+
+cd $OBJROOT/lnd/obj
+
+set comp = 'unknown'
+if ($COMP_INTERFACE == 'MCT' ) set comp = mct
+if ($COMP_INTERFACE == 'ESMF') set comp = esmf
+
+cat >! Filepath << EOF
+$CASEROOT/SourceMods/src.vic
+$CODEROOT/lnd/vic/vic
+$CODEROOT/lnd/vic/vic/vic_run/src
+$CODEROOT/lnd/vic/vic/vic_run/include
+$CODEROOT/lnd/vic/vic/drivers/shared/src
+$CODEROOT/lnd/vic/vic/drivers/shared/include
+$CODEROOT/lnd/vic/vic/drivers/cesm/src
+$CODEROOT/lnd/vic/vic/drivers/cesm/include
+$CODEROOT/lnd/vic/vic/drivers/cesm/cpl_$comp/
+EOF
+
+set vicdefs = ""
```

Comment:
Turns out this was pretty simple, the change was just
```xml
<compiler COMPILER="intel">
  ...
  <ADD_CFLAGS MODEL="vic">  -std=c99 </ADD_CFLAGS>
  <ADD_CFLAGS MODEL="vic" DEBUG="TRUE"> -Wall -traceback -DLOG_LVL=0 </ADD_CFLAGS>
</compiler>
```
yielding a this in the Macros file:
```csh
ifeq ($(DEBUG), TRUE) 
  ifeq ($(MODEL), vic) 
     CFLAGS += -Wall -traceback -DLOG_LVL=0 
  endif

   CFLAGS += -g 
   FFLAGS += -g -CU -check pointers -check bounds -fpe0 -ftz 
endif

...

ifeq ($(MODEL), vic) 
   CFLAGS +=  -std=c99 
endif
```
pretty slick I'd say.